source/DayPicker.js

```diff
@@ -0,0 +1,149 @@
+(function (enyo, scope) {
+	/**
+	* {@link moon.DayPicker}, which extends {@link moon.ExpandablePicker}, is
+	* a drop-down picker menu that solicits day of the week from the user. 
+	*
+	* ```
+	* {kind: 'moon.DayPicker'}
+	* ```
+	*
+	* When the picker is minimized, the currently selected day are
+	* displayed as subtext below the picker label. And if the picker select every weekday,
+	* subtext will be changed 'Every Weekday' automatically.
+	*
+	* The content of representative value can be changed.
+	* ```
+	* {kind: 'moon.DayPicker', everyWeekday:'Weekdays', everyWeekend:'Weekends', everyday:'Daily'}
+	* ```
+	*
+	* @class moon.DayPicker
+	* @extends moon.ExpandablePicker
+	* @ui
+	* @public
+	*/
+	enyo.kind(
+		/** @lends moon.DayPicker.prototype */ {
+
+		/**
+		* @private
+		*/
+		name: 'moon.DayPicker',
+
+		/**
+		* @private
+		*/
+		kind: 'moon.ExpandablePicker',
+
+		/**
+		* @private
+		*/
+		autoCollapseOnSelect: false,
+
+		/**
+		* @private
+		*/
+		multipleSelection: true,
+
+		/**
+		* @private
+		*/
+		content: moon.$L('Select days'), 
+
+		/**
+		* @private
+		* @lends moon.DayPicker.prototype
+		*/
+		published: {
+
+			/**
+			* Text to be displayed when all of the day are selected.
+			*
+			* @type {String}
+			* @default 'Everyday'
+			* @public
+			*/
+			everydayText: moon.$L('Everyday'),
+
+			/**
+			* Text to be displayed when all of the weekday are selected.
+			*
+			* @type {String}
+			* @default 'Every Weekday'
+			* @public
+			*/
+			everyWeekdayText: moon.$L('Every Weekday'),
+
+			/**
+			* Text to be displayed when all of the weekend are selected.
+			*
+			* @type {String}
+			* @default 'Every Weekend'
+			* @public
+			*/
+			everyWeekendText: moon.$L('Every Weekend'),
+
+			/**
+			* Text to be displayed as the current value if no item is currently selected.
+			*
+			* @type {String}
+			* @default 'Nothing selected'
+			* @public
+			*/
+			noneText: moon.$L('Nothing selected')
+		},
+
+		/**
+		* @private
+		*/
+		dayOfTheWeek: [
```

Comment:
Yeah I'm wondering if, at this point, we shouldn't just make the requisite changes in this PR that we had broke out into ENYO-930 since this PR is still open and it might be better to fully complete the work in one cohesive PR. We had originally broken it into separate tickets because we had wanted to resolve ENYO-519 for the sprint, but that didn't end up happening.